README.md

```diff
@@ -257,4 +257,21 @@ docker::exec { 'helloworld-uptime':
   tty       => true,
 }
 ```
+### Docker-compose
 
+Docker-compose allows you to configure and run multple containers from a single erb file.
+The Puppet module expects to find the docker-compose.yml file first.
+```puppet
+file { '/root/docker-compose.yml':
+  ensure    => file,
+  content   => template('helloworld/docker-compose.yml.erb'), 
+  }
+```
+Then will build the containers with the following statement
+
+```puppet
+docker_compose {'Hello World':
+  source   => '/root',  
+  require  => File['/root/docker-compose.yml']
```

Comment:
Rather than always require people to do this manually it might be nicer to do this using autorequire in the type (eg. https://github.com/puppetlabs/puppetlabs-aws/blob/master/lib/puppet/type/ec2_instance.rb#L235-L237) and make source an absolute reference to the file?